gulpfile.js

```diff
@@ -34,88 +34,90 @@ const babel = require('gulp-babel');
+				if (err) return cb(err);
+				cb();
+			});
+		}
+	});
 }
 
 function babelify() {
-    return gulp.src('./build/safari.safariextension/zotero/**/*.js')
-        .pipe(buffer())
-        .pipe(plumber({errorHandler: onError}))
-        .pipe(babel({
-            presets: ['es2015']
-        }))
-        .pipe(gulp.dest((data) => data.base));
+	return gulp.src([
+		'./build/safari.safariextension/**/*.js',
+	])
+		.pipe(buffer())
+		.pipe(plumber({errorHandler: onError}))
+		.pipe(babel({
+			presets: ['es2015']
+		}))
+		.pipe(gulp.dest((data) => data.base));
 }
 
 function minify() {
-	return gulp.src('./build/**/*.js')
+	return gulp.src('./build/bookmarklet/**/*.js')
 		.pipe(buffer())
 		.pipe(babel({
-            presets: ['babili'],
-            comments: false
-        }))
+			presets: ['babili'],
+			comments: false
+		}))
 		.pipe(gulp.dest((data) => data.base));
 }
 
-gulp.task('watch', function() {
-    gulp.watch(['./src/chrome/**', './src/common/**', './src/safari/**'], ['build-dev']);
+gulp.task('watch', function () {
+	gulp.watch(['./src/chrome/**', './src/common/**', './src/safari/**'], ['build-dev']);
 });
 
-gulp.task('watch-chrome', function() {
-    gulp.watch(['./src/chrome/**', './src/common/**', './src/safari/**'], ['build-dev', 'reloadChromeExtensionsTab']);
+gulp.task('watch-chrome', function () {
+	gulp.watch(['./src/chrome/**', './src/common/**', './src/safari/**'], ['build-dev', 'reloadChromeExtensionsTab']);
 });
 
 gulp.task('reloadChromeExtensionsTab', ['build-dev', 'safarify'], reloadChromeExtensionsTab);
 
-gulp.task('build-sh-dev', function(cb) {
-    var proc = spawn('./build.sh', ['-d'], {cwd: process.cwd()});
-    proc.stdout.on('data', function(data) {
-        process.stdout.write(data);
-    });
-    proc.stderr.on('data', function(data) {
-        process.stderr.write(data);
-    });
-    proc.on('close', function(code) {
-        cb(code)
-    })
+gulp.task('build-sh-dev', function (cb) {
+	var proc = spawn('./build.sh', ['-d'], {cwd: process.cwd()});
+	proc.stdout.on('data', function (data) {
+		process.stdout.write(data);
+	});
+	proc.stderr.on('data', function (data) {
+		process.stderr.write(data);
+	});
+	proc.on('close', function (code) {
+		cb(code)
+	})
 });
 
-gulp.task('build-sh', function(cb) {
-    var proc = spawn('./build.sh', [], {cwd: process.cwd()});
-    proc.stdout.on('data', function(data) {
-        process.stdout.write(data);
-    });
-    proc.stderr.on('data', function(data) {
-        process.stderr.write(data);
-    });
-    proc.on('close', function(code) {
-        cb(code)
-    })
+gulp.task('build-sh', function (cb) {
+	var proc = spawn('./build.sh', [], {cwd: process.cwd()});
+	proc.stdout.on('data', function (data) {
+		process.stdout.write(data);
+	});
+	proc.stderr.on('data', function (data) {
+		process.stderr.write(data);
+	});
+	proc.on('close', function (code) {
+		cb(code)
+	})
```

Comment:
Most/all of the above should just be squashed into https://github.com/zotero/zotero-connectors/pull/51/commits/b4f2aaa0db3fd5a49374bfe248e3565c0970a76c.